lib/iris/fileformats/pp_rules.py

```diff
@@ -902,6 +905,19 @@ def _all_other_rules(f):
             (AuxCoord('son', long_name='season', units='no_unit'),
              None))
 
+    # Special case where year is zero and months match.
+    # Month coordinates (--> scalar coordinates)
+    if (f.lbtim.ib == 2 and f.lbtim.ic in [1, 2, 4] and
+            ((len(f.lbcode) != 5) or
+             (len(f.lbcode) == 5 and
+              f.lbcode.ix not in [20, 21, 22, 23] and
+              f.lbcode.iy not in [20, 21, 22, 23])) and
+            f.lbyr == 0 and f.lbyrd == 0 and
+            f.lbmon == f.lbmond):
+        aux_coords_and_dims.append(
+            (AuxCoord(f.lbmon, long_name='month', units='no_unit'),
+             None))
+
```

Comment:
In this case, it is correct (in my view) not to have a time or forecast_reference_time coordinate.  I think the forcast period may still valid and useful, in general.  
@djkirkham specifically in the files you are looking at, what is the 
LBFT
value, is it ever anything other than 0?